tools-local/unix/cmake/get-version.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+usage()
+{
+    echo "usage: $0 <repository-root> cmake <override-scripts-folder-path> <strict-tool-version-match> <tool-path>"
+    echo "  repository-root: This argument is ignored."
+    echo "  override-scripts-folder-path: This argument is ignored."
+    echo "  strict-tool-version-match: This argument is ignored."
+    echo "  tool-path: Path to CMake executable or the folder containing the executable."
+    echo "If successful then, returns the version number of CMake executable."
+    echo "Exit 1 if the executable is not available at the specified path or folder."
+}
+
+if [ $# -ne 5 ]; then
+    usage
+    exit 1
+fi
+
+toolName="$2"
+toolPath="$5"
+
+if [ "$toolName" != "cmake" ]; then
+    echo "Second argument should be cmake."
+    usage
+    exit 1
+fi
+
+if [ ! -f "$toolPath" ]; then
+    "Argument specified as tool-path does not exist or is not accessible. Path: $toolPath"
```

Comment:
Good catch. Added.

---
In reply to: [111203635](https://github.com/ravimeda/corefx/pull/9#discussion_r111203635) [](ancestors = 111203635)